docs/spec/v1beta1/provider.md

```diff
@@ -8,81 +8,85 @@ Spec:
 
 ```go
 type ProviderSpec struct {
-	// Type of provider
-	// +kubebuilder:validation:Enum=slack;discord;msteams;rocket;generic;github;gitlab;bitbucket;azuredevops;googlechat;webex;sentry;azureeventhub;telegram;lark;matrix;opsgenie
-	// +required
-	Type string `json:"type"`
-
-	// Alert channel for this provider
-	// +optional
-	Channel string `json:"channel,omitempty"`
-
-	// Bot username for this provider
-	// +optional
-	Username string `json:"username,omitempty"`
-
-	// HTTP/S webhook address of this provider
-	// +kubebuilder:validation:Pattern="^(http|https)://"
-	// +optional
-	Address string `json:"address,omitempty"`
-
-	// HTTP/S address of the proxy
-	// +kubebuilder:validation:Pattern="^(http|https)://"
-	// +optional
-	Proxy string `json:"proxy,omitempty"`
-
-	// Secret reference containing the provider webhook URL
-	// +optional
-	SecretRef *meta.LocalObjectReference `json:"secretRef,omitempty"`
-
-	// CertSecretRef can be given the name of a secret containing
-	// a PEM-encoded CA certificate (`caFile`)
-	// +optional
```

Comment:
Please undo this change, Go formatting is not optional for us.

Reply:
sorry - my vscode auto-formats optimistically